### ISSUE TITLE
Long RUN

### ISSUE THREAD

**Issue by ocgabs** [ocgabs/Caribbean#22]
Val re-started on ARCHER2 from 2009 and is currently on 2011...

**Comment by vleguenn** [comment]
I tried to re-built all the rest of the available NEMO outputs (/work/n01/n01/valegu/CARIBBEAN_NEMO_4_0_4/nemo/cfgs/Caribbean/EXP_FullOcean/OUTPUTS/). However, I managed to rebuild only outputs with a time frequency greater than 5d. Using the REBUILD_NEMO tool, I would rebuild using : ./rebuild_nemo.exe. But now, when using either 1h,1d or 25h, it gets killed. (It used to work fine before). So I am wondering if I need to use a script instead. I asked James if he had one...

@mpayopayo, do you manage to rebuild outputs with the frequency 1h, 1d or 25h? 

I also had space issues, so I deleted all outputs with `_0000`, `_0001`, `_0002` and` _0003` extension (after verifying with `ncview` that the rebuild output was not corrupted). The long run without rivers is currently at year 2015 (pending).

**Comment by mpayopayo** [comment]
yes, @vleguenn  same here. They keep getting killed when trying to rebuild. I tried reloading the modules needed for building nemo in case there was something missing but it doesn't change.

**Comment by mpayopayo** [comment]
problem with rebuilding frequency >5d sorted here: https://github.com/ocgabs/Caribbean/issues/20
